backend/src/main/java/com/htecgroup/skynest/repository/UserObjectAccessRepository.java

```diff
@@ -4,5 +4,10 @@
 import com.htecgroup.skynest.model.entity.UserObjectAccessKey;
 import org.springframework.data.repository.CrudRepository;
 
+import java.util.List;
+import java.util.UUID;
+
 public interface UserObjectAccessRepository
-    extends CrudRepository<UserObjectAccessEntity, UserObjectAccessKey> {}
+    extends CrudRepository<UserObjectAccessEntity, UserObjectAccessKey> {
+  List<UserObjectAccessEntity> findAllByObjectId(UUID bucketId);
```

Comment:
It doesn't make sense for the method to be called findAllByObjectId and to recieve parametar named bucketId